src/rm.js

```diff
@@ -149,6 +149,11 @@ function _rm(options, files) {
     if (stats.isDirectory() && options.recursive) {
       rmdirSyncRecursive(file, options.force);
     }
+
+    // Path is a symbolic link
+    if (stats.isSymbolicLink()) {
```

Comment:
Maybe we should turn this into an else-if ladder?

Also, what happens if this is a different type of file, such as a fifo (like `mkfifo` creates)?